bigquery/query.rb

```diff
@@ -31,22 +31,28 @@ def unique_words project_id
         # [END build_service]
 
         # [START run_query]
-        sql = "SELECT TOP(corpus, 10) as title, COUNT(*) as unique_words " +
-              "FROM [publicdata:samples.shakespeare]"
         results = bigquery.query sql
         # [END run_query]
 
         # [START print_results]
-        results.each do |row|
-          puts "#{row['title']}: #{row['unique_words']}"
+        results.each_with_index do |row, index|
+          puts "--- Row #{index+1} ---"
+          for column, value in row
+            puts "#{column}: #{value}"
```

Comment:
What are column and value?

In the previous sample, column data was printed out using explicit names.  That really helps me understand what is being printed as a user.  The query probably SELECTs a few columns and I'd love to see those columns printed out explicitly, eg. `row["title"]`